apps/ledger-live-mobile/src/newArch/components/QueuedDrawer/index.tsx

```diff
@@ -81,14 +81,17 @@ const QueuedDrawer = ({
     setIsDisplayed(true);
   }, []);
 
+  const onCloseRef = useRef(onClose);
+  onCloseRef.current = onClose;
+
   const triggerClose = useCallback(() => {
     setIsDisplayed(false);
     if (drawerInQueueRef.current?.getPositionInQueue() !== 0) {
       drawerInQueueRef.current?.removeDrawerFromQueue();
       drawerInQueueRef.current = undefined;
     }
-    onClose && onClose();
-  }, [onClose]);
+    onCloseRef.current && onCloseRef.current(); // hack to avoid triggering the useEffect below if the parent changes the onClose prop
+  }, []);
```

Comment:
No problem with the hack here, just pointing out that they recently talked about `useEvent` for this kind of scenario
https://github.com/reactjs/rfcs/blob/useevent/text/0000-useevent.md
https://github.com/facebook/react/issues/14099

Reply:
aaah interesting !